supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/runnable/RunnableBuilder.scala

```diff
@@ -232,7 +232,7 @@ object RunnableBuilder extends BatchUtilityConversions {
         case (key, value) if key == Key.Tag => Key.Logging -> value
         case (key, value) => key -> value
       }
-    ).withTimeout(timeout = 300.seconds)
+    ).withTimeout(timeout = 30.minutes) // Consider removing this timeout altogether, other runnables don't have them.
```

Comment:
It would be good to know what the default is – perhaps 7 days, same as the user runnable.

If so, we probably don't want a totally hung runnable to spend that much money.

Reply:
My interpretation of [the docs](https://cloud.google.com/batch/docs/set-timeouts) is that the default timeout is none, and that if no runnable-level timeout is set, Batch just cares about the overall task-level timeout, which is that 7-day one (configurable in Cromwell).